lib/dry/component/container.rb

```diff
@@ -143,6 +143,14 @@ def self.require_component(key, &block)
           Kernel.require component.path
           yield(component.constant) if block
         else
+          imports.each do |ns, container|
+            begin
+              localized_key = key.sub(/^#{Regexp.escape(ns)}#{Regexp.escape(config.namespace_separator)}/, '')
+              return container.require_component(localized_key, &block)
+            rescue ArgumentError
+            end
+          end
+
```

Comment:
This looks like we're missing some API. Would it be possible to ask a container whether a dependency exists or not?